app/src/features/FhirAttributePanel/Condition.tsx

```diff
@@ -0,0 +1,250 @@
+          }).unwrap();
+        }
+    };
+
+    createOrUpdateCondition();
+  }, [condition, createCondition, dispatch, updateCondition]);
+
+  const handleColumnChange = async (column: PendingColumn) => {
+    if (column.id) {
+      dispatch(columnUpdated({ id: column.id, changes: column }));
+    }
+
+    if (column.id && column.table && column.column && column.owner) {
+      const newColumn = column.pending
+        ? await createColumn({
+            columnRequest: column as ColumnRequest,
+          }).unwrap()
+        : await updateColumn({
+            id: column.id,
+            columnRequest: column as ColumnRequest,
+          }).unwrap();
+
+      dispatch(
+        columnUpdated({
+          id: column.id,
+          changes: { ...newColumn, pending: false },
+        })
+      );
+
+      if (condition.id && newColumn.id !== condition.column) {
+        dispatch(
+          conditionUpdated({
+            id: condition.id,
+            changes: { column: newColumn.id },
+          })
+        );
+      }
+    }
+  };
+  const handleDeleteCondition = async () => {
+    !condition.pending &&
+      condition.id &&
+      (await deleteCondition({ id: condition.id }));
+    condition.id && dispatch(conditionRemoved(condition.id));
+  };
+  const handleRelationChange = (
+    event: React.ChangeEvent<{ name?: string | undefined; value: unknown }>
+  ) => {
+    if (condition.id)
+      dispatch(
+        conditionUpdated({
+          id: condition.id,
+          changes: { relation: event.target.value as ConditionRelationEnum },
+        })
+      );
+  };
+
+  const handleValueChange = (
+    event: React.ChangeEvent<{ name?: string | undefined; value: unknown }>
+  ) => {
+    if (condition.id)
+      dispatch(
+        conditionUpdated({
+          id: condition.id,
+          changes: { value: event.target.value as string | undefined },
+        })
+      );
+  };
+
+  return (
+    <Grid
+      item
+      container
+      alignItems="center"
+      spacing={1}
+      justify="space-between"
+    >
+      <Grid container item xs={11} spacing={1} alignItems="center">
+        <Grid item>
+          <Typography className={classes.badgeLabel}>
+            {t("useThisGroupIf")}
+          </Typography>
+        </Grid>
+        <ColumnSelects
+          pendingColumn={conditionColumn ?? {}}
+          onChange={handleColumnChange}
+        />
+        <Grid item>
+          <Select
+            value={condition.relation ?? ""}
+            options={CONDITION_RELATIONS.map((relation) => ({
+              id: relation,
+              label: t(relation),
+            }))}
+            onChange={handleRelationChange}
+            emptyOption={t("selectOperation")}
+          />
+        </Grid>
+        <Grid item>
+          <TextField
+            value={condition.value ?? ""}
+            onChange={handleValueChange}
+            placeholder={t("typeValue")}
+            variant="outlined"
+            size="small"
+          />
```

Comment:
is this displayed if the relation is "IS_NULL" or "IS_NOT_NULL" ?

Reply:
Yes

Reply:
I asked myself if I should remove it if that's the case
I should then add conditions to reset `value` to an empty string whenever `relation` is set to `NULL` or `NOT_NULL`

Reply:
yes I thin it's worth adding it